components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityMgtConstants.java

```diff
@@ -26,6 +26,16 @@ private IdentityMgtConstants(){}
     public static final String USER_IDENTITY_CLAIMS = "UserIdentityClaims";
     public static final String ERROR_CODE_DELIMITER = "-";
 
+    /**
+     * User account states.
+     */
+    public class AccountStates {
+
+        private AccountStates() {}
+
+        public static final String PENDING_ADMIN_FORCED_USER_PASSWORD_RESET = "PENDING_FUPR";
```

Comment:
Any specific reasons to add this constant in IdentityMgtConstants class? We already have PENDING constants in IdentityRecoveryConstants class https://github.com/wso2-extensions/identity-governance/blob/master/components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java#L106